README.md

```diff
@@ -78,37 +78,75 @@ PyMilo is an open source Python package that provides a simple, efficient, and s
 
 
 ## Usage
-### Model preparation 
+Imagine you want to train a `LinearRegression` model representing this equation: `y = 1 * x_0 + 2 * x_1 + 3`. You make your data points (`X`, `y`) and train your model as follows.
```

Comment:
1- I suggest using this: $y = x_{0}+2x_{1}+3$ instead of `y = 1 * x_0 + 2 * x_1 + 3`
2- JSON file instead of `.json` file

@sadrasabouri

Reply:
You'll create data points (`X`, `y`) and train the model as follows.